src/app/shared/refresh.service.ts

```diff
@@ -139,4 +139,28 @@ export class RefreshService {
     this.tools.push(tool);
     this.containerService.setTools(this.tools);
   }
+
+  syncServices(): void {
+    const message = 'Syncing services';
+    this.alertService.start(message);
+    this.usersService.syncUserServices().subscribe(
+      response => {
+        this.alertService.detailedSuccess();
+        this.workflowService.setWorkflows(response);
```

Comment:
Rename response to something more descriptive and add type.

Reply:
Fixed by making a common function as you suggested below.